angular-poller.js

```diff
@@ -308,6 +308,19 @@
                 reset: function () {
                     this.stopAll();
                     pollers = [];
+                },
+
+                /**
+                * Stop and removes a specific poller service
+                */
+                remove: function (poller) {
```

Comment:
Why passing in `poller` here instead of adding a `delete` function to `Poller.prototype`? That way you can do something like this in your controller:
```javascript
myPoller.delete();
```